.travis.yml

```diff
@@ -18,7 +18,7 @@ install: beaver install
 
 script:
     - beaver make USE_BLAS=openblas USE_OPENCV=0
-    - beaver run ./build-tsunami-package
+    - BEAVER_DOCKER_VARS="F" beaver run ./build-tsunami-package
```

Comment:
So the script does not currently use the build flavor but in future you might want to create a debug package? For that you would include then F=devel in the build matrix. What about making it more explicit like

```
- beaver make USE_BLAS=openblas USE_OPENCV=0 F=production
- beaver run ./build-tsunami-package
- beaver make USE_BLAS=openblas USE_OPENCV=0 F=devel
- beaver run ./build-tsunami-package
```

meaning having one build creating both packages. What is different in a debug package?

Reply:
If we're _going_ to do both development and production builds, they should be kept cleanly separated.  Otherwise we risk having accidents where a development build of `libmxnet.{a,so}` wind up in package marked for production.

But we aren't, right now.  I just raise it as a motivation for ensuring that `F` gets passed correctly to the `build-tsunami-package` script.

Reply:
It is a bit unfortunate that you cannot call `beaver make pkg` directly but have to this additional setup. Because essentially the script forward to `make pkg`.

Reply:
We could probably rework things a bit at some point (feel free to try out alternatives).  But for now, let's just fix the immediate point of concern.

Reply:
Just to make sure: `F` is the only variable you want to pass on?

Reply:
That I'm aware of.  We certainly don't want to pass on things like the D compiler (since we don't use it).

AFAIK `DIST` is not used by Makd.  We use `lsb_release` directly when checking the distro name in `build-tsunami-package`, for example.